specs/README.md

```diff
@@ -1,84 +1,164 @@
+| Design Representative | Member of the Windows API Design Representatives group responsible for the shape and direction of their component area within Windows or Project Reunion |
+| Reviewer              | Person providing commentary on an API under review                                                                                                       |
 
-* Create unsolicited PRs that add or modify  specs. To propose a new feature
-    please follow the Project Reunion contribution process [described here](../docs/contributor-guide.md)
-    and start by [opening a new issue](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+For additional help, create a new `api-design` tagged issue and assign it to @jonwis for assistance.
 
-## Spec workflow
+### Authoring an API Spec
 
 For Project Reunion team members, the spec and API design workflow is:
 
 1. Ensure your proposal is approved and in plan.
 
-2. Create a new working branch ```user/<username>/<feature>``` or work in
-   your fork of the Project Reunion repo.
+2. Create a new working branch `user/<username>/<feature>` or work in your fork of the Project
+   Reunion repo.
+
+3. Create a new folder for your spec under the `specs` folder: `/specs/<feature>`
+
+4. Author your spec using the [spec template](spec_template.md). Please use relative links for
+   images or other assets in the folder. Be sure to read and follow the commented instructions in
+   the template.
+
+5. Work with your API design representative and iterate on the specification in your branch as much
+   as you see fit.
+
+When your spec is ready for review, your API representative will help guide you through the
+remainder.
+
+**DO** follow the authoring guidelines from [the contributor's guide](../docs/contributor-guide.md).
+
+**DO** follow the spec template guidance for authoring samples and IDL.
+
+**DO NOT** create a pull request from `user/<username>/<feature>` to `main` (see "Reviewing"
+section).
+
+### Reviewing APIs
+
+Your API representative will help you finish the design document and get it reviewed with members of
+the Windows API Design community. The process to finish the review is:
+
+1. The representative creates a new pull request from `user/<username>/<feature>` to the `main`
+   branch. Make sure the pull request mentions the original issue thread.
+
+2. The representative schedules time with the Windows API Design community to review the pull
+   request.
+
+    - As the design group is spread out we prefer giving no less than 48h notice when reviewing APIs
+    - Reviews are typically scheduled for Tuesdays 1-2pm PST/PDT or Thursdays 1-3pm PST/PDT
+    - API reviewers should post in the initial issue thread with the date and time of the review
+
+3. When scheduling a review, representatives should _only_ look at the _first_ commit in the pull
+   request. API authors should _avoid_ pushing new commits to the pull request while review is
+   happening.
```

Comment:
I suspect this rule may have a chilling effect on owners trying to fulfill Reunion principles by working in the open. Consider the scenario in which the owner opens a draft PR, performs a bunch of work in preparation for API review (along with the community), and is now ready to submit for Formal API Review™. The careful owner, skimming the rules again, sees Rule 3 and now has to make a weird choice:
* Squash all their atomic commits / rewrite history
    * Oops: Any user dependent on that PR (e.g. via fork, branch validation, co-contributor, etc.) is now in a broken state and has to perform a few Git rituals before they can contribute again
* Close the draft PR and re-open a new one with a single commit

I propose instead that the review representatives look at all changed files, as of the _last_ commit submitted for review (or last commit that made it in before the scheduled review window). The owner can be asked to provide a set of changes, up to a particular commit, by tacking on that last commit to the Files Changed URL, e.g. `microsoft/ProjectReunion/pull/541/files/42e2f9e7255aa9a2fe41083af83c56b948c1ca53`. This can alternatively be accomplished via the Changes from X Commit dropdown that after clicking the Commits tab.)

Reply:
Good point. The current description is my faulty remembrance of how we did this for a previous review, where instead what happened was someone made a branch and merged from the `user/...` branch into it as a single commit.

I'll use the notifications review as an example.